.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report (Beta)
+about: Report a Bug or an Issue with Slimefun 4.
+title: ''
+labels: ['\U0001F3AF Needs testing', '\U0001F41E Bug Report']
+assignees: ''
+body:
+- type: markdown
+  attributes:
+    value: '## Thanks for reporting a bug!'
+- type: textarea
+  label: ':round_pushpin: Description'
+  validations:
+    required: true
+  description: |
+    A clear and detailed description of what went wrong.
+    The more information you can provide, the easier we can handle this problem.
+- type: textarea
+  label: ':bookmark_tabs: Reproduction Steps'
+  validations:
+    required: true
+  description: |
+    Tell us the exact steps to reproduce this issue, the more detailed the easier we can reproduce it.
+    Youtube Videos and Screenshots are recommended!
+  value: |
+    1.
+    2.
+    3.
+- type: textarea
+  label: ':bulb: Expected Behavior'
+  validations:
+    required: true
+  description: |
+    What were you expecting to happen?
+    What do you think would have been the correct behaviour?
+- type: textarea
+  label: ':scroll: Server Log'
+  description: |
+    Take a look at your Server Log and post any errors you can find via https://pastebin.com/
+    If you are unsure about it, post your full log, you can find it under /logs/latest.log
+- type: textarea
```

Comment:
Is there a way to do a file upload here?

Reply:
All textarea support file upload by default

Reply:
Alright, I would really need to see this in action first I guess